### PR TITLE
Feature/end game

### PR DESCRIPTION
**Pull request by gauthierboeshertz**

### CODE REVIEW

File: app/src/androidTest/java/sdp/moneyrun/MenuActivityTest.java [hunk 1]
```diff
@@ -86,14 +86,13 @@ public void mapButtonAndSplashScreenWorks() {
             Intents.init();
 
             Espresso.onView(withId(R.id.map_button)).perform(ViewActions.click());
-            Thread.sleep(1000);
```
Comment (Alvaro2112) [inline app/src/androidTest/java/sdp/moneyrun/MenuActivityTest.java:89]: Why was this removed?

**Review by Alvaro2112 — CHANGES_REQUESTED** [review]

File: app/src/androidTest/java/sdp/moneyrun/MenuActivityTest.java (hunk 1 above)
Comment (gauthierboeshertz) [reply app/src/androidTest/java/sdp/moneyrun/MenuActivityTest.java:89]: It was useless

Comment (gauthierboeshertz) [reply app/src/androidTest/java/sdp/moneyrun/MenuActivityTest.java:89]: It was a mistake nice catch Alvaro!

**Review by Alvaro2112 — APPROVED** [review]

File: app/src/main/java/sdp/moneyrun/EndGameActivity.java [hunk 2]
```diff
@@ -13,7 +16,7 @@
  */
```
Comment (Tesa27) [inline app/src/main/java/sdp/moneyrun/EndGameActivity.java:16]: Even though the methods are straight forward and understandable at first sight, a general documentation 1-2 sentences could be useful in the future or in other classes

File: app/src/main/java/sdp/moneyrun/EndGameActivity.java [hunk 3]
```diff
@@ -1,6 +1,9 @@
 package sdp.moneyrun;
```
Comment (Tesa27) [inline app/src/main/java/sdp/moneyrun/EndGameActivity.java:1]: Also a suggestion, the functionality right now includes displaying the score and a button to go back to main, maybe we would like to restart a game without needing to go to main menu and rejoin a game. If all the players want to play the "same" game again they should have an option to do so, but this could be a task to follow and to ponder if it makes sense.

**Review by Tesa27 — APPROVED** [review]
